lib/portal.js

```diff
@@ -9,6 +9,27 @@ const KEYCODES = {
 
 export default class Portal extends React.Component {
 
+  static defaultProps = {
+    onOpen: () => {},
+    onClose: () => {},
+    onUpdate: () => {},
+  };
+
+  static propTypes = {
+    className: React.PropTypes.string,
+    style: React.PropTypes.object,
+    children: React.PropTypes.element.isRequired,
+    target: React.PropTypes.instanceOf(window.HTMLElement),
```

Comment:
IMHO would be better if you remove `window` of this propType.
It won't work for components that use `Portal` and are rendered in the server side.
I tried to use your fork on my project and got an error because of it
```
  target: _react2.default.PropTypes.instanceOf(window.HTMLElement),
                                               ^

ReferenceError: window is not defined
```